edxval/api.py

```diff
@@ -143,6 +143,53 @@ def update_video_status(edx_video_id, status):
     video.save()
 
 
+def get_transcript_credentials_state_for_org(org, provider=None):
+    """
+    Returns transcript credentials state for an org
+
+    Arguments:
+        org (unicode): course organization
+        provider (unicode): transcript provider
+
+    Returns:
+        dict: provider name and their credential existance map
+
+        {
+            u'Cielo24': True
+        }
+        {
+            u'3PlayMedia': False,
+            u'Cielo24': True
+        }
+    """
+    credentials_state = {}
+    query_filter = {'org': org}
+    if provider:
+        query_filter['provider'] = provider
+
+    try:
+        credentials_state = {
+            credential.provider: credential.exists
+            for credential in ThirdPartyTranscriptCredentialsState.objects.filter(**query_filter)
+        }
+    except ThirdPartyTranscriptCredentialsState.DoesNotExist:
```

Comment:
filter won't raise `ThirdPartyTranscriptCredentialsState.DoesNotExist`. We are good without `try: except..` here.

I think just return the following would work.
```bash
          {
              credential.provider: credential.exists
              for credential in ThirdPartyTranscriptCredentialsState.objects.filter(**query_filter)
         }
```